challenges/functions.js

```diff
@@ -7,24 +7,40 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+let consume = function(a, b, cb){
+  return cb(a,b);
+}
+
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+let add = function(a,b){
+  console.log( a + b);
+}
 
+let multiply =function(a,b){
+  console.log(a * b);
+}
+
+let greeting = function(firstName, lastName, cb){
+  console.log(firstName + " " + lastName + ' nice to meet you!');
+}
 
 /* Step 3: Check your work by uncommenting the following calls to consume(): */
-//consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
+consume(2,2,add); // 4
```

Comment:
If you want these to show you'd have to return the results inside of your functions.